src/components/Nav/Nav.js

```diff
@@ -1,22 +1,22 @@
 import React from 'react';
-import './Nav.module.css';
+import style from './Nav.css';
```

Comment:
You can get rid of "style from".

Reply:
![image](https://user-images.githubusercontent.com/34000503/101088513-7ea81300-35bc-11eb-8ed7-12563a0ead3e.png)